### DISCUSSION TITLE
Would I be able to compile this to WASM?

### DISCUSSION THREAD

**Discussion by NovaAndrom3da** [ptitSeb/box64#252] [Q&A]
First, I have to say- this project is amazing and it works very well.

I would like to ask, is it possible to use emscripten to compile box86 & box64 to WebAssembly without any extensive modifications? If so, this will be very helpful. Thanks!

**Comment by ptitSeb** [comment]
I haven't tried to build with emscripten. An interpretor only build is probably doable as emscripten is quite posix compatible. Not sure about some of the most tricky function on signals.c tho.
